### PR TITLE
HW2: rebased and squashed the commits below

### PR DESCRIPTION
**Pull request by Cargo1284**
added markdown for my project proposal and a project that I want to work on

fixed bullet points in my proposal and added link titles to links in my contribution

moved my projects into a folder and chnged the file name for them

added the proposal and contribution links to cmg2249.md

### CODE REVIEW

File: Projects/Python/FantasyBaseballTrends.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Fantasy Baseball Lineup Helper
```
Comment (timkpaine) [inline Projects/Python/FantasyBaseballTrends.md:1]: Add project link

File: students/cmg2249.md [hunk 2]
```diff
@@ -5,20 +5,11 @@ Name: Carlos Gonzalez-Romero
...
+GitHub README: [link](https://github.com/Cargo1284/Cargo1284/blob/main/README.md)
 
 ## Project Proposal
-
-1. Fantasy baseball daily stats reports for your team
-2. College football webscraper that tells you information on the projected closest games
-3. Baseball pitcher matchup report
-4. Budget Tracker
-5. To do list
+- [FantasyBaseballTrends.md](./projects/python/FantasyBaseballTrends.md)
```
Comment (timkpaine) [inline students/cmg2249.md:11]: update link path and re-squash 😈

File: students/cmg2249.md [hunk 3]
```diff
@@ -5,20 +5,11 @@ Name: Carlos Gonzalez-Romero
...
 
 ## Project Contributions
+- [openlibrary.md](./projects/python/openlibrary.md)
```
Comment (timkpaine) [inline students/cmg2249.md:14]: update link path and re-squash 😈

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: Projects/Python/FantasyBaseballTrends.md (hunk 1 above)
Comment (Cargo1284) [reply Projects/Python/FantasyBaseballTrends.md:1]: I addressed the issues in a new pull request with the branch hw2_branch-2.

**Comment by Cargo1284** [comment]
I addressed all of the issues in another pull request because I messed this branch up trying to fix it.

**Pull request closed without merging**